automation/test.sh

```diff
@@ -1,18 +1,39 @@
-#!/bin/bash -e
+#!/bin/bash -ex
 
 main() {
-    ls -l
     install_promtool
+    install_yq
+
+    echo "Extract Prometheus rules"
+    ./yq ".spec" prometheus/base/prometheus.rules.yaml > test/promql/extracted-rules.yaml
```

Comment:
@yftacherzog
As this also relevant to pint, I suggest moving it outside of test.sh. WDYT?

Reply:
I think taking it out of this script means creating dependencies between tasks.  You'd either need to push this into git - which I don't want (I'm adding those yamls to `.gitignore`) or making them run in sequence somehow sharing the same disk, which seems more of a burden than a gain.

The easiest approach would be to have the linter run as part of this script as well, but I'm not sure we want that.

Reply:
I created another PR on top of this one makes this script specific to Prometheus and provide it with multiple options that will allow it to prepare once and run the relevant test on different executions. Have a look.